contrib/builder/rpm/generate.sh

```diff
@@ -96,5 +109,10 @@ for version in "${versions[@]}"; do
 
 	echo 'ENV AUTO_GOPATH 1' >> "$version/Dockerfile"
 
-	echo 'ENV DOCKER_BUILDTAGS selinux' >> "$version/Dockerfile"
+	DOCKER_BUILDTAGS="selinux"
+	if [[ "$distro" == "amazon" ]]; then
+		DOCKER_BUILDTAGS="$DOCKER_BUILDTAGS !journald"
```

Comment:
good call then we can avoid adding a new image :)

On Mon, Nov 2, 2015 at 3:42 PM, Avi Miller <notifications@github.com> wrote:

> In contrib/builder/rpm/generate.sh
> <https://github.com/docker/docker/pull/17283#discussion_r43700123>:
>
> > @@ -96,5 +109,10 @@ for version in "${versions[@]}"; do
> >
> >  	echo 'ENV AUTO_GOPATH 1' >> "$version/Dockerfile"
> >
> > -	echo 'ENV DOCKER_BUILDTAGS selinux' >> "$version/Dockerfile"
> > +	DOCKER_BUILDTAGS="selinux"
> > +	if [[ "$distro" == "amazon" ]]; then
> > +		DOCKER_BUILDTAGS="$DOCKER_BUILDTAGS !journald"
>
> Have you considering using the Oracle Linux 6 (FROM oraclelinux:6) image
> instead? That matches the Amazon Linux AMI far more closely than CentOS 7.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/17283/files#r43700123>.
>